lib/src/core/better_player_controller.dart

```diff
@@ -1062,47 +1113,36 @@ class BetterPlayerController {
       throw StateError("The data source has not been initialized");
     }
 
-    final bool isPipSupported =
-        (await videoPlayerController!.isPictureInPictureSupported()) ?? false;
+    final playerContext = betterPlayerGlobalKey.currentContext;
+    final RenderBox? renderBox = playerContext?.findRenderObject() as RenderBox?;
+    if (renderBox == null) {
+      BetterPlayerUtils.log(
+          "Can't show PiP. RenderBox is null. Did you provide valid global"
+          " key?");
+      return;
+    }
+    final Offset position = renderBox.localToGlobal(Offset.zero);
 
-    if (isPipSupported) {
-      final playerContext = betterPlayerGlobalKey.currentContext!;
-      final RenderBox? renderBox =
-          playerContext.findRenderObject() as RenderBox?;
-      if (renderBox == null) {
-        BetterPlayerUtils.log(
-            "Can't show PiP. RenderBox is null. Did you provide valid global"
-            " key?");
-        return;
-      }
-      final Offset position = renderBox.localToGlobal(Offset.zero);
-
-      if (Platform.isAndroid) {
-        await videoPlayerController?.enablePictureInPicture(
-          left: position.dx,
-          top: position.dy,
-          width: renderBox.size.width,
-          height: renderBox.size.height,
-        );
-        return;
-      }
-      if (Platform.isIOS) {
-        await videoPlayerController?.enablePictureInPicture(
-          left: position.dx,
-          top: position.dy,
-          width: renderBox.size.width,
-          height: renderBox.size.height,
-        );
-        _postEvent(BetterPlayerEvent(BetterPlayerEventType.pipStart));
-        return;
-      } else {
-        BetterPlayerUtils.log("Unsupported PiP in current platform.");
-      }
+    if (Platform.isAndroid) {
+      await videoPlayerController?.enablePictureInPicture(
+        left: position.dx,
+        top: position.dy,
+        width: renderBox.size.width,
+        height: renderBox.size.height,
+      );
+      return;
+    }
+    if (Platform.isIOS) {
+      await videoPlayerController?.enablePictureInPicture(
+        left: position.dx,
+        top: position.dy,
+        width: renderBox.size.width,
+        height: renderBox.size.height,
+      );
+      _postEvent(BetterPlayerEvent(BetterPlayerEventType.pipStart));
```

Comment:
is this missing event `BetterPlayerEventType.pipStart` for android?

Reply:
also, seems android /iOS are now the same

Reply:
event này em đang để nó bắn từ `onPictureInPictureStatusChanged` trong thằng `android/src/main/kotlin/uz/shs/better_player_plus/BetterPlayer.kt` cho chuẩn anh ạ

Reply:
```dart

await videoPlayerController?.enablePictureInPicture(
  left: position.dx,
  top: position.dy,
  width: renderBox.size.width,
  height: renderBox.size.height,
);
if (Platform.isIOS) {
   _postEvent(BetterPlayerEvent(BetterPlayerEventType.pipStart));
}
```